packages/notifi-dapp-example/next.config.mjs

```diff
@@ -2,29 +2,30 @@
 import CopyPlugin from "copy-webpack-plugin";
 
 const nextConfig = {
-  output: 'export' /** BUILD ONLY */,
+  // output: 'export' /** BUILD ONLY */,
   images: {
     formats: ['image/webp'],
   },
-  typescript: {
-    // !! WARN !!
-    // Dangerously allow production builds to successfully complete even if
-    // your project has type errors.
-    // !! WARN !!
-    ignoreBuildErrors: true,
-  },
-  eslint: {
-    // Warning: This allows production builds to successfully complete even if
-    // your project has ESLint errors.
-    ignoreDuringBuilds: true,
-  },
+  // typescript: { /** BUILD ONLY */
+  //   // !! WARN !!
+  //   // Dangerously allow production builds to successfully complete even if
+  //   // your project has type errors.
+  //   // !! WARN !!
+  //   ignoreBuildErrors: true,
+  // },
+  // eslint: { /** BUILD ONLY */
+  //   // Warning: This allows production builds to successfully complete even if
+  //   // your project has ESLint errors.
+  //   ignoreDuringBuilds: true,
+  // },
```

Comment:
Rather than using comment, we can dynamic define the build or dev. Here is the example, hope it helps: https://github.com/notifi-network/notifi-sdk-ts/blob/main/packages/notifi-react-example-v2/next.config.mjs